Header header
string ssid
string mac
int64 rss
